app/react/kubernetes/applications/components/ConfigurationsFormSection/ConfigMapsFormSection.tsx

```diff
@@ -62,7 +62,7 @@ export function ConfigMapsFormSection({
             configurations={configMaps}
             onRemoveItem={() => onRemoveItem(index)}
             index={index}
-            dataCyType="config"
+            configurationType="ConfigMap"
```

Comment:
do we need to change the `dataCyType` here?

Reply:
Yeah, the text (configmap or secret) is now used for a texttip as well as the dataCy for the Select component. Because it's used for a couple of things, I renamed it more generally